Content.Server/Xenoarchaeology/Equipment/ArtifactAnalyzerSystem.cs

```diff
@@ -86,7 +86,7 @@ private float GetGlimmerMultiplier(ArtifactAnalyzerComponent comp)
     {
         float normalizedGlimmer = _glimmerSystem.Glimmer / 1000f;
```

Comment:
Let's just make super-sure we don't take sqrt of a negative

```suggestion
        float normalizedGlimmer = Math.Clamp(_glimmerSystem.Glimmer / 1000f, 0, 1);
```